### PR TITLE
Document starting with an observer node and moving to a maintenance node

### PR DESCRIPTION
**Pull request by timothymcmackin**
Update to cover starting with an observer node and moving to a maintenance mode.

Preview: https://docs-etherlink-git-operator-upgrade-trili-tech.vercel.app/network/smart-rollup-nodes

### CODE REVIEW

File: docs/network/smart-rollup-nodes.md [hunk 1]
```diff
@@ -12,16 +12,137 @@ The data for the original kernel is stored in separate files called _preimages_.
...
+   :::
+
+- An account with at least 10,000 liquid (unstaked) tez, referred to as the _operator account_.
+You can use the same account that you use for a layer 1 baker, but for better security, you can use a different account and delegate its tez to the layer 1 account.
+
+Follow these steps to convert a Smart Rollup node from observer mode to operator mode:
+
+1. Stop the Smart Rollup node.
+
+1.
```
Comment (lthms) [inline docs/network/smart-rollup-nodes.md:139]:
```suggestion
1. Stop the Smart Rollup node.
```

? feels not finished

File: docs/network/smart-rollup-nodes.md [hunk 2]
```diff
@@ -12,16 +12,137 @@ The data for the original kernel is stored in separate files called _preimages_.
...
+
+
+
+
+
 
-- [Running the Smart Rollup node in operator mode](#running-the-smart-rollup-node-in-operator-mode)
-- [Running the Smart Rollup node in observer mode](#running-the-smart-rollup-node-in-observer-mode)
 
 ## Running the Smart Rollup node in operator mode
```
Comment (lthms) [inline docs/network/smart-rollup-nodes.md:147]: You probably want to remove this section all together, right?

**Review by lthms — CHANGES_REQUESTED** [review]
I am under the impression that you added me as a reviewer a bit too early in the process?

**Comment by timothymcmackin** [comment]
> I am under the impression that you added me as a reviewer a bit too early in the process?

Sorry, I jumped the gun. It should be ready now.

File: docs/network/smart-rollup-nodes.md [hunk 3]
```diff
@@ -57,144 +47,134 @@ The best place to get the most recent binary files to use with Etherlink is http
...
-1. Initialize the local context of the node, which is where it stores local data:
+   :::warning Use a private layer 1 node
 
-   1. Set the environment variable `SR_DATA_DIR` to the directory where the node should store its local data.
-   The default value is `$HOME/.tezos-smart-rollup-node`.
+   To run the Smart Rollup node in a mode that posts commitments, including maintenance mode and operator mode, you must connect it to a layer 1 node that you control.
+   Using a public layer 1 node as the basis for a Smart Rollup node in maintenance or operator mode exposes the Smart Rollup node to security risks.
+   For example, a malicious layer 1 node can expose the Smart Rollup node to an incorrect branch, which can cause the Smart Rollup node to post invalid commitments and lose tez when other node operators computing from the correct branch refute them.
 
-   1. Initialize the local context by running this command and using the address of the Etherlink Smart Rollup, which you can get from the [Network information](/get-started/network-information) page:
```
Comment (lthms) [inline docs/network/smart-rollup-nodes.md:144]: I would advise to (1) bootstrap on a public node if required on observer mode, (2) switch to your own node still in observer mode, (3) assess the rollup node keep operating as expected, then (4) switch to maintenance mode.

**Comment by lthms** [comment]
This is a really neat document, filling a gap for our partners and adopters. Thanks a tone for it! just left a small comment.

File: docs/network/smart-rollup-nodes.md (hunk 3 above)
Comment (timothymcmackin) [reply docs/network/smart-rollup-nodes.md:144]: Got it, reworked the order to switch to your node first and then go to maintenance mode.

**Review by lthms — APPROVED** [review]

**Pull request merged**